packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -331,50 +331,52 @@ class MinimumTextContrastGuideline extends AccessibilityGuideline {
     if (shouldSkipNode(data)) {
       return result;
     }
+    final String text = data.label.isEmpty ? data.value : data.label;
+    final List<Element> elements = find.text(text).hitTestable().evaluate().toList();
+    for (final Element element in elements) {
+      result += await _evaluateElement(node, element, tester, image, byteData);
+    }
+    return result;
+  }
 
+  Future<Evaluation> _evaluateElement(
+      SemanticsNode node,
+      Element element,
+      WidgetTester tester,
+      ui.Image image,
+      ByteData byteData,
+      ) async {
+    const Evaluation result = Evaluation.pass();
```

Comment:
What is this extra const variable for? Can you just use `const Evaluation.pass()` in place of `result` below to clear up the code?

Reply:
Updated.